README.md

```diff
@@ -90,6 +90,32 @@ an error when attempting to image the machine.
 </a>
 </p>
 
+## Building from source
+
+Building Restor from source is _not_ required for general usage. Please see the
+[Releases](https://github.com/google/restor/releases) page to download a
+pre-compiled version of Restor.
+
+#### Requirements
+
+* Xcode 9+ installed
+* [cocoapods](https://cocoapods.org) installed
+* A valid "Mac Developer" Signing Certificate from Apple
+* Xcode command line tools installed
+
+#### Build steps
+
+1. `git clone https://github.com/google/restor.git`
+1. `cd restor`
+1. `pod install`
+1. Build with the following command, making sure to insert a valid Team Identifier.
+    ```bash
+    make google_release TEAM_ID=EQHXZ8M8AV
```

Comment:
Change google_release to release to match my other comment.

The Team ID can be found using:
`security find-certificate -p -c "Mac Developer" | openssl x509 -inform pem -subject | perl -ne '/OU=(\w+)\// && print $1'`